hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/fsdataset/impl/FsDatasetImpl.java

```diff
@@ -3811,5 +3818,10 @@ void stopAllDataxceiverThreads(FsVolumeImpl volume) {
   public List<FsVolumeImpl> getVolumeList() {
     return volumes.getVolumes();
   }
+
+  @Override
+  public void updateLastDirScannerFinishTime(long time) {
```

Comment:
nit: can we rename to `setLastDirScannerFinishTime`, following the getter/setter convention?

Reply:
@xinglin Sir, thanks for suggestions.  Have changed.